.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,115 @@
+---
+name: Bug report
+description: Is something not working? Help us fix it!
```

Sounds way more collaborative.
```suggestion
description: Is something not working? Let's fix it together!
```